<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>553</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>O3PRM Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsrc/resources.qrc">
    <normaloff>:/icons/logo</normaloff>:/icons/logo</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>665</width>
     <height>408</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="projectExplorator">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QWidget" name="dockLayout" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="128,0">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="documentLayout" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="128">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="margin">
             <number>1</number>
            </property>
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="elideMode">
               <enum>Qt::ElideRight</enum>
              </property>
              <property name="documentMode">
               <bool>true</bool>
              </property>
              <property name="tabsClosable">
               <bool>true</bool>
              </property>
              <property name="movable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QStackedWidget" name="dockStack">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pageSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTreeWidget" name="searchDock">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="headerHidden">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageBuild">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="buildDock">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="spacing">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageExecute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="execDock"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTabBar" name="tabBar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionO3PRM_File"/>
     <addaction name="actionRequest_File"/>
     <addaction name="separator"/>
     <addaction name="actionPackage"/>
     <addaction name="separator"/>
     <addaction name="actionNewProject"/>
    </widget>
    <widget class="QMenu" name="menuRecentProjects">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Recent Projects</string>
     </property>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="actionOpenProject"/>
    <addaction name="menuRecentProjects"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuFind">
     <property name="title">
      <string>Find</string>
     </property>
     <addaction name="actionFind"/>
     <addaction name="actionFindNext"/>
     <addaction name="actionFindPrevious"/>
    </widget>
    <widget class="QMenu" name="menuFont">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionSelect_Font"/>
     <addaction name="separator"/>
     <addaction name="actionBigger"/>
     <addaction name="actionSmaller"/>
     <addaction name="actionDefaultSize"/>
    </widget>
    <widget class="QMenu" name="menuGlobal_Settings">
     <property name="title">
      <string>Global Settings</string>
     </property>
     <addaction name="actionShowLineNumbers"/>
     <addaction name="actionShowBookmarks"/>
     <addaction name="actionShowInvisibleCharacters"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="menuFind"/>
    <addaction name="separator"/>
    <addaction name="actionAutocomplete"/>
    <addaction name="separator"/>
    <addaction name="actionComment"/>
    <addaction name="actionUncomment"/>
    <addaction name="separator"/>
    <addaction name="actionIncreaseIndentation"/>
    <addaction name="actionDecreaseIndentation"/>
    <addaction name="separator"/>
    <addaction name="menuFont"/>
    <addaction name="menuGlobal_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShowToolbar"/>
    <addaction name="actionShowStatusBar"/>
    <addaction name="actionShowProjectExplorator"/>
    <addaction name="separator"/>
    <addaction name="actionShowSearch"/>
    <addaction name="separator"/>
    <addaction name="actionShowBuild"/>
    <addaction name="actionShowExecute"/>
   </widget>
   <widget class="QMenu" name="buildMenu">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="separator"/>
    <addaction name="actionExecute"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="buildMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionO3PRM_File">
   <property name="text">
    <string>O3PRM File</string>
   </property>
  </action>
  <action name="actionRequest_File">
   <property name="text">
    <string>Request File</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionPackage">
   <property name="text">
    <string>Package</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionAutocomplete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Autocompletion</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="actionUncomment">
   <property name="text">
    <string>Uncomment</string>
   </property>
  </action>
  <action name="actionIncreaseIndentation">
   <property name="text">
    <string>Increase Indentation</string>
   </property>
  </action>
  <action name="actionDecreaseIndentation">
   <property name="text">
    <string>Decrease Indentation</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionFindPrevious">
   <property name="text">
    <string>Find Previous</string>
   </property>
  </action>
  <action name="actionUseSelectionForFind">
   <property name="text">
    <string>Use Selection for Find</string>
   </property>
  </action>
  <action name="actionSelect_Font">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select Font</string>
   </property>
  </action>
  <action name="actionBigger">
   <property name="text">
    <string>Bigger</string>
   </property>
  </action>
  <action name="actionSmaller">
   <property name="text">
    <string>Smaller</string>
   </property>
  </action>
  <action name="actionDefaultSize">
   <property name="text">
    <string>Default Size</string>
   </property>
  </action>
  <action name="actionShowLineNumbers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Line Numbers</string>
   </property>
  </action>
  <action name="actionShowBookmarks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bookmarks</string>
   </property>
  </action>
  <action name="actionShowInvisibleCharacters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Invisible Characters</string>
   </property>
  </action>
  <action name="actionShowToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionShowProjectExplorator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Project Explorator</string>
   </property>
  </action>
  <action name="actionShowSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionShowBuild">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionShowExecute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>QTabBar</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rsrc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
